gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt

```diff
@@ -23,7 +23,8 @@ internal object ComposeCompilerCompatibility {
         "1.9.20" to "1.5.3",
         "1.9.21" to "1.5.4",
         "1.9.22" to "1.5.8.1",
-        "1.9.23" to "1.5.10.2",
+        "1.9.23" to "1.5.13",
```

Comment:
It looks suspicious. Usually one version of compose compiler supports only one version of kotlin.
Or did you disable the version checker in the Compose Compiler plugin code?

Reply:
you are definitely right, fixed